### PR TITLE
LEAR-1317: cursor based pagination while getting all messages of a room

### PR DESCRIPTION
**Pull request by Bivek10**
ref:[jira](https://readytowork.atlassian.net/browse/LEAR-1317)

### CODE REVIEW

File: api/controllers/message.go [hunk 1]
```diff
@@ -77,11 +79,13 @@ func (mc MessageController) FindAll(c *gin.Context) {
 		responses.HandleError(c, err)
 		return
 	}
-	sd, err := mc.messageService.FindAll(roomId)
+	sd, nextCursor, err := mc.messageService.FindAll(pagination, roomId)
```
Comment (Damodar009) [inline api/controllers/message.go:82]: @Bivek10 change sd to messages

**Review by neekesh — APPROVED** [review]
LGTM

**Review by stha-nikhil — APPROVED** [review]
LGMT :fire:

**Review by Damodar009 — APPROVED** [review]
LGTM

**Pull request merged**